lib/writer.js

```diff
@@ -147,3 +162,44 @@ function currentDate() {
 
   return util.format('%d-%s-%s', now.getFullYear(), pad(now.getMonth() + 1), pad(now.getDate()));
 }
+
+function nonGithubUrl(url) {
```

Comment:
It doesn't actually fix non github url. https://github.com/npm/npm/blob/master/lib/repo.js#L69 fixes it.